File: freq.js
function freqSeq(str, sep) {
    let arr = str.split('')
    let newO = {}, value;
    for (let i = 0; i< arr.length; i++){
      value = arr[i];
      if (value in newO){
        newO[value]++;
      }else{
        newO[value] = 1;
      }
    }
    let newArr = arr.map(x=>newO[x]);
    return newArr.join(`${sep}`)
  }
  

File: freq.py
def freq_seq(s, sep):
    arr = [*s]
    freq = {}
    for x in arr:
        if x in freq:
            freq[x] += 1
        else:
            freq[x] = 1
    newArr = [freq.get(x, '')for x in arr]
    newList = [str(x) for x in newArr]
    return sep.join(newList)
    